public/polyfill.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-var */
+
+// Test in case we are in safari private mode which fails on any storage
+try {
+	window.localStorage.setItem( 'localStorageTest', '' );
+	window.localStorage.removeItem( 'localStorageTest' );
+} catch ( error ) {
+	var _data = {};
+
+	// We can't replace window.localStorage, but we can replace its methods
+	window.localStorage.setItem = function( id, val ) {
```

Comment:
since we're not minifying this, you could stash `window.localStorage` into a `var s` and assign against that. Save a few bytes. :) 

Though maybe gzip eats it too.